### PR TITLE
Enable ssh agent forwarding

### PR DESCRIPTION
**Pull request by dpoirier**
Allow deploying users to forward their SSH agents.

### CODE REVIEW

**Review by copelco — APPROVED** [review]
Nice! Worked for me on a deploy. I like this approach better than #14.

**Comment by dpoirier** [comment]
Maybe I should back out the 'subproject' changes from this PR so I can go ahead and merge the ssh-agent changes while we're reaching consensus on ``subproject``?

**Comment by copelco** [comment]
Agreed, sounds good

File: README.rst [hunk 1]
```diff
@@ -68,6 +73,7 @@ The following variables are made use of by the ``tequila-common``
 role:
 
 - ``project_name`` **required**
+- ``project_user`` **default:** ``{{ project_name }}``
```
Comment (vkurup) [inline README.rst:76]: This line is duplicated a few lines below.

File: tasks/main.yml [hunk 2]
```diff
@@ -77,28 +123,28 @@
...
 
-- name: disable user accounts that are not listed devs or unmanaged accounts
+- name: disable user accounts that are not listed devs, unmanaged accounts, or project users
   user:
     name: "{{ item }}"
     state: absent
   with_items: "{{ current_users | difference(active_users) | difference(unmanaged_users) }}"
   vars:
-    current_users: "{{ current_users_out.stdout_lines | difference([project_user]) }}"
+    current_users: "{{ current_users_out.stdout_lines | difference([project_user])  }}"
```
Comment (vkurup) [inline tasks/main.yml]: Nitpick: looks like an extra space snuck in, making it look like this line changed.

**Review by vkurup — APPROVED** [review]
A couple minor nitpicks, but this looks good. Appreciate all the comments explaining 'why' and 'how'.

**Pull request merged**